projects/kserve/testing/config.yaml

```diff
@@ -224,57 +226,62 @@ ci_presets:
   # multi-model
   # ---
 
-  watsonx_multi_model_gating:
-    extends: [gating, 5min, e2e_gpu, e2e_multi, raw, watsonx_mutlti_model_models]
-    matbench.lts.opensearch.index: rhoai-watsonx-multi
-    matbench.lts.horreum.test_name: rhoai-watsonx-multi
+  cpt_multi_model_gating:
+    extends: [gating, 5min, e2e_gpu, e2e_multi, raw, cpt_multi_model_models]
+    matbench.lts.opensearch.index: rhoai-kserve-multi
+    matbench.lts.horreum.test_name: rhoai-kserve-multi
     tests.e2e.limits_equals_requests: false
     tests.e2e.llm_load_test.args.concurrency: 16
 
-  watsonx_multi_model_models:
+  cpt_multi_model_models:
     extends: [tgis, raw, use_intlab_minio]
+    model_directory_prefix: cpt
```

Comment:
@npalaska, you need:
1. to create the field in the main configuration, before updating it here in the presets (you cannot update a field that doesn't exist, that's a current limitation of the python code handling the configuration)
2. please have this setting in a more logical place, not at the top level (`kserve.model` seems to be the right place)